app/models/ems_refresh/save_inventory_physical_infra.rb

```diff
@@ -52,18 +52,18 @@ def save_physical_servers_inventory(ems, hashes, target = nil)
     store_ids_for_new_records(ems.physical_servers, hashes, :ems_ref)
   end
 
-  def save_configuration_templates_inventory(ems, hashes, target = nil)
+  def save_customization_scripts(ems, hashes, target = nil)
```

Comment:
This method should have `_inventory` on the end.  Otherwise I don't think it will be called.